src/test/kotlin/org/jitsi/videobridge/health/config/HealthConfigTest.kt

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright @ 2018 - present 8x8, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jitsi.videobridge.health.config
+
+import org.jitsi.config.BooleanMockConfigValueGenerator
+import org.jitsi.config.DurationToLongMockConfigValueGenerator
+import org.jitsi.config.LongMockConfigValueGenerator
+import org.jitsi.config.runBasicTests
+import org.jitsi.videobridge.JitsiConfigTest
+
+class HealthConfigTest : JitsiConfigTest() {
+    init {
+        "health interval" {
+            runBasicTests(
+                legacyConfigName = "org.jitsi.videobridge.health.INTERVAL",
```

Comment:
I don't understand the purpose of this test. The internals of the config code itself are already tested in jicoco, so essentially what the test does is compare the strings in `HealthIntervalProperty` to the string literals here, i.e. if forces us to keep two synchronized copies of the strings. Am I missing something?

Reply:
Yeah, this is part of the reason why I originally thought we could get a way without tests for these properties.  You're not wrong: the health config properties don't make for the most compelling test.  But we do verify that the property class is properly wired in to both configs and is checking them in the correct priority order.  That being said, you'll notice I didn't do tests for the other ones here, largely because it didn't feel all that necessary (though we always can).  Things get a bit more interesting with the test for the websocket config (which I have in another branch and will put in a follow-up PR).

As far as the strings, I agree it's annoying to have to copy them.  I have some ideas on how to improve that, but it'd be when I bring back the config validation (which I've begun playing with).